src/views/Home/components/EarnAssetCard.tsx

```diff
@@ -31,17 +32,21 @@ const latestPools: Pool[] = orderBy(activeNonCakePools, ['sortOrder', 'pid'], ['
 const assets = ['CAKE', ...latestPools.map((pool) => pool.earningToken.symbol)].join(', ')
 
 const EarnAssetCard = () => {
+  const { t } = useTranslation()
+  const assetText = t('Earn %assets% in Pools', { assets })
+  const assetTextParts = assetText.split(assets)
```

Comment:
Same here, let's deconstruct for readability.

Reply:
Done